lang/en/admin/setting.php

```diff
@@ -0,0 +1,143 @@
+        'app_favicon_help' => 'Favicons should be placed in the public folder, located in the root panel directory.',
+        'debug_mode' => 'Debug Mode',
+        'navigation' => 'Navigation',
+        'sidebar' => 'Sidebar',
+        'topbar' => 'Topbar',
+        'unit_prefix' => 'Unit Prefix',
+        'decimal_prefix' => 'Decimal Prefix (MB/GB)',
+        'binary_prefix' => 'Binary Prefix (MiB/GiB)',
+        '2fa_requirement' => '2FA Requirement',
+        'not_required' => 'Not Required',
+        'admins_only' => 'Required for Admins Only',
+        'all_users' => 'Required for All Users',
+        'trusted_proxies' => 'Trusted Proxies',
+        'trusted_proxies_help' => 'New IP or IP Range',
+        'clear' => 'Clear',
+        'set_to_cf' => 'Set to Cloudflare IPs',
+        'display_width' => 'Display Width',
+    ],
+    'captcha' => [
+        'enable' => 'Enable Turnstile Captcha?',
+        'info' => '<p>You can generate the keys on your <u><a href="https://developers.cloudflare.com/turnstile/get-started/#get-a-sitekey-and-secret-key" target="_blank">Cloudflare Dashboard</a></u>. A Cloudflare account is required.</p>',
+        'site_key' => 'Site Key',
+        'secret_key' => 'Secret Key',
+        'verify' => 'Verify Domain?',
+    ],
+    'mail' => [
+        'mail_driver' => 'Mail Driver',
+        'test_mail' => 'Send Test Mail',
+        'test_mail_sent' => 'Test Mail sent',
+        'test_mail_failed' => 'Test Mail failed',
+        'from_settings' => 'From Settings',
+        'from_settings_help' => 'Set the Address and Name used as "From" in mails.',
+        'from_address' => 'From Address',
+        'from_name' => 'From Name',
+        'smtp' => [
+            'smtp_title' => 'SMTP Configuration',
+            'host' => 'Host',
+            'port' => 'Port',
+            'username' => 'Username',
+            'password' => 'Password',
+            'encryption' => 'Encryption',
+            'ssl' => 'SSL',
+            'tls' => 'TLS',
+            'none' => 'None',
+        ],
+        'mailgun' => [
+            'mailgun_title' => 'Mailgun Configuration',
+            'domain' => 'Domain',
+            'secret' => 'Secret',
+            'endpoint' => 'Endpoint',
+        ],
+    ],
+    'backup' => [
+        'backup_driver' => 'Backup Driver',
+        'throttle' => 'Throttles',
+        'throttle_help' => 'Configure how many backups can be created in a period. Set period to 0 to disable this throttle.',
+        'limit' => 'Limit',
+        'period' => 'Period',
+        'seconds' => 'Seconds',
+        's3' => [
+            's3_title' => 'S3 Configuration',
+            'default_region' => 'Default Region',
+            'access_key' => 'Access Key ID',
+            'secret_key' => 'Secret Access Key',
+            'bucket' => 'Bucket',
+            'endpoint' => 'Endpoint',
+            'use_path_style_endpoint' => 'Use Path Style Endpoint',
+        ],
+    ],
+    'oauth' => [  //to do All the helper text
```

Comment:
Can't have comments in the translation files
```suggestion
    'oauth' => [
```

Reply:
... you know it was there for a reason, and that reason is I forgot to go back and do them 😄